modules/rbac-permission-sets.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: CONCEPT
+[id="rbac-permission-sets_{context}"]
+= System permission sets
+
+[role="_abstract"]
+{product-title} includes some default system permission sets that you can apply on users.
```

Comment:
"apply on users" sounds odd, maybe "apply to users"?

Reply:
I've corrected this to `apply to roles`.